app/containers/StudyPage/StudyPage.tsx

```diff
@@ -484,7 +501,7 @@ class StudyPage extends React.Component<StudyPageProps, StudyPageState> {
                             {this.renderBackButton(
                               site.siteView,
                               '⤺︎ Back',
-                              '/search',
+                              `/search/${this.props.match.params.searchId}`,
```

Comment:
Make the back button adhere to the URL rather than user's history in the /search link